tests/foreman/cli/test_activationkey.py

```diff
@@ -183,7 +183,7 @@ def test_positive_create_with_usage_limit_default_explicitly(self):
 
         @Assert: Activation key is created
         """
-        new_ak = self._make_activation_key({u'unlimited-content-hosts': '1'})
+        new_ak = self._make_activation_key()
```

Comment:
Emm, because it isn't supported anymore and I verify that by default value `content-host-limit` put into `Unlimited`.

All I can do is to rename test into `create_with_usage_limit_default`, but I doubt it is a blocking one change =)